app/src/main/java/zapsolutions/zap/channelManagement/ManageChannelsActivity.java

```diff
@@ -164,6 +166,14 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
                 openChannelBSDFragment.show(getSupportFragmentManager(), OpenChannelBSDFragment.TAG);
             }
         }
+
+        if (requestCode == REQUEST_CODE_OPEN_CHANNEL && resultCode == ScanNodePubKeyActivity.RESULT_CODE_LNURL_CHANNEL) {
+            if (data != null) {
+                LnUrlChannelResponse channelResponse = (LnUrlChannelResponse) data.getSerializableExtra(ScanNodePubKeyActivity.EXTRA_CHANNEL_RESPONSE);
+                LnUrlChannelBSDFragment lnUrlChannelBSDFragment = LnUrlChannelBSDFragment.createLnURLChannelDialog(channelResponse);
+                lnUrlChannelBSDFragment.show(getSupportFragmentManager(), "lnurlChannelBottomSheetDialog");
```

Comment:
Would use the `TAG` variable here as well

Reply:
:+1: 
I will change it.